dashboards/ansible-tower/ansible-tower.json

```diff
@@ -0,0 +1,806 @@
+{
```

Comment:
I noticed some entity guids and account ids in here, let's get those cleaned up before merge. The `sanitize-dashboard` utility can automatically do that for you, docs are [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards)

Reply:
awesome.. thank you @aswanson-nr . Dashboard is now sanitised. 😃